nightwatch/test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+cd $(dirname $0)
+
+ARGS="$@"
+[ -z "$ARGS" ] && ARGS='--env default'
+
+if [[ "$ARGS" != *"--env default"* ]] ; then
+    CRED="../mycredentials"
+    [ ! -f $CRED ] && echo "Could not find credentials file: $CRED" && exit 1
```

Comment:
you could do the same as the `runtests.sh` and touch the file if it doesn't exist anyway